drgn/helpers/linux/net.py

```diff
@@ -13,17 +13,30 @@
 from typing import Iterator, Union
 
 from drgn import NULL, IntegerLike, Object, Program
-from drgn.helpers.linux.list import hlist_for_each_entry
+from drgn.helpers.linux.list import hlist_for_each_entry, list_for_each_entry
 from drgn.helpers.linux.list_nulls import hlist_nulls_for_each_entry
 
 __all__ = (
+    "for_each_netns",
     "netdev_get_by_index",
     "netdev_get_by_name",
     "sk_fullsock",
     "sk_nulls_for_each",
 )
 
 
+def for_each_netns(prog: Program) -> Iterator[Object]:
```

Comment:
It looks like in the kernel, this is called `for_each_net`. I try to stick to the kernel naming (except when it's confusing or too generic, like `dev_get_by_{index,name}`), so would you mind renaming it to `for_each_net`?